lib/ts/recipe/thirdparty/api/implementation.ts

```diff
@@ -194,13 +194,14 @@ export default function getAPIInterface(): APIInterface {
         },
 
         appleRedirectHandlerPOST: async function ({ code, state, options }): Promise<void> {
-            const redirectURL =
-                options.appInfo.websiteDomain.getAsStringDangerous() +
-                options.appInfo.websiteBasePath.getAsStringDangerous() +
-                "/callback/apple?state=" +
-                state +
-                "&code=" +
-                code;
+            // const redirectURL =
+            //     options.appInfo.websiteDomain.getAsStringDangerous() +
+            //     options.appInfo.websiteBasePath.getAsStringDangerous() +
+            //     "/callback/apple?state=" +
+            //     state +
+            //     "&code=" +
+            //     code;
+            const redirectURL = "" + code + state;
```

Comment:
Revert this, also this is extremely bad practice

If you are temporarily commenting code make sure to add a proper TODO explaining that you need to revert this